ui/components/app/confirm/info/row/address.test.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { CHAIN_IDS } from '../../../../../../shared/constants/network';
+import mockState from '../../../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../../../test/lib/render-helpers';
+import { shortenAddress } from '../../../../../helpers/utils/util';
+import configureStore from '../../../../../store/store';
+import { ConfirmInfoRowAddress } from './address';
+import { TEST_ADDRESS } from './constants';
+
+const render = (
+  storeOverrides: Record<string, any> = {},
+  address: string = TEST_ADDRESS,
+) => {
+  const store = configureStore({
+    metamask: { ...mockState.metamask },
+    ...storeOverrides,
+  });
+
+  return renderWithProvider(<ConfirmInfoRowAddress address={address} />, store);
+};
+
+describe('ConfirmInfoRowAddress', () => {
+  it('should render appropriately with PetNames enabled', () => {
```

Comment:
I don't think this is a necessary change, more of a heads up - we're steering away from beginning test phrases with "should" 

> Do not use "should" at the beginning of the test name. 
https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test